b2/src/test/java/org/jclouds/b2/internal/BaseB2ApiLiveTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.labs.b2.internal;
+
+import java.util.Properties;
+
+import org.jclouds.apis.ApiMetadata;
+import org.jclouds.apis.BaseApiLiveTest;
+import org.jclouds.labs.b2.B2Api;
+import org.jclouds.labs.b2.B2ApiMetadata;
+
+import com.google.inject.Injector;
+import com.google.inject.Module;
+
+public class BaseB2ApiLiveTest extends BaseApiLiveTest<B2Api> {
+
+   protected BaseB2ApiLiveTest() {
+      provider = "b2";
+   }
+
+   @Override
+   protected ApiMetadata createApiMetadata() {
+      return new B2ApiMetadata();
+   }
+
+   @Override
+   protected B2Api create(Properties props, Iterable<Module> modules) {
+      Injector injector = newBuilder().modules(modules).overrides(props).buildInjector();
+      return injector.getInstance(B2Api.class);
+   }
```

Comment:
Is there a real need to override this method? It looks like it does the same than the parent?